Makefile

```
@@ -14,6 +14,8 @@ BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 
 ifneq ($(origin OUTPUT_DIR), undefined)
 BUNDLE_OUTPUT_DIR = --output-dir=$(OUTPUT_DIR)
+else
+OUTPUT_DIR = bundle
 endif
```

There is a syntax for setting a value if `undefined`: `?=`
Using it you can re-write this section like this:

```suggestion
OUTPUT_DIR ?= bundle
BUNDLE_FLAGS = --output-dir=$(OUTPUT_DIR)
```